launch/test/counter.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import argparse
+import signal
+import sys
+import time
+
+
+def main(argv=sys.argv[1:]):
+    parser = argparse.ArgumentParser(
+        description=(
+            'Simple program outputting a counter. '
+            'Even values go to STDERR, odd values go to STDOUT.'))
```

Comment:
@dirk-thomas if it is for style reasons, that I suppose that is ok, but it's not technically needed.

@esteve I do not believe you need the `\`. As long as it is in the containing `()` it is safe, and if you remove the `()` it would become a syntax error.